api/Models/Location/Location.cs

```diff
@@ -10,6 +10,8 @@ public class Location
         public string Name { get; set; }
         public string Code { get; set; }
         public string LocationId { get; set; }
+        public string JustinLocationName { get; set; }
+        public string JustinLocationId { get; set; }
```

Comment:
This seems to be imposing some conventions from PCSS.  These values from PCSS should map into the existing properties.  If you look at the data after it's mapped you should see the duplication.